### PR TITLE
Fix: create only default chain accounts for vault wallet

### PR DESCRIPTION
**Pull request by Asmadek**
* Fix: create only default chain accounts for vault wallet
* fix: decode single frame dd address response
* exclude base polkadot vault account from balance subscription

### CODE REVIEW

**Comment by Asmadek** [comment]
closes #1412

**Comment by pgolovkin** [comment]
I like this solution!

File: src/renderer/pages/Onboarding/Vault/ManageVault/model/manage-vault-model.ts [hunk 1]
```diff
@@ -103,8 +101,14 @@ sample({
 
 sample({
   clock: formInitiated,
-  fn: () => {
-    return chains.reduce<DraftAccount<ChainAccount | ShardAccount>[]>((acc, chain) => {
+  source: {
```
[the remark this answers is not in the document]
Comment (Asmadek) [reply src/renderer/pages/Onboarding/Vault/ManageVault/model/manage-vault-model.ts]: I fix it, but I like such approach, to make feature diff more clear if we will modify this source again.

File: src/renderer/pages/Onboarding/Vault/ManageVault/model/manage-vault-model.ts [hunk 2]
```diff
@@ -103,8 +101,14 @@ sample({
...
+    chains: networkModel.$chains,
+  },
+  fn: ({ chains }) => {
+    const chainList = Object.values(chains);
+    const defaultChains = chainList.filter((chain) => isDefaultChain(chain.chainId));
```
[the remark this answers is not in the document]
Comment (Asmadek) [reply src/renderer/pages/Onboarding/Vault/ManageVault/model/manage-vault-model.ts]: Made it, but we additionally use Westend for dev build, and if we move it in one day from third place current code will be broken

**Review by tuul-wq — APPROVED** [review]

**Pull request merged**